kaldo/forceconstants.py

```diff
@@ -126,76 +118,32 @@ def from_folder(cls,
                     is_acoustic_sum: bool = False,
                     only_second: bool = False,
                     distance_threshold: float | None = None,
-                    chunk_size: int = 100000):
-        """
-        Create a finite difference object from a folder
-
-        The folder should contain the a set of files whose names and contents are dependent on the "format" parameter.
-        Below is the list required for each format (also found in the api_forceconstants documentation if you prefer
-        to read it with nicer formatting and explanations).
-
-        - numpy: replicated_atoms.xyz, second.npy, third.npz
-        - eskm: CONFIG, replicated_atoms.xyz, Dyn.form, THIRD
-        - lammps: replicated_atoms.xyz, Dyn.form, THIRD
-        - shengbte: CONTROL, POSCAR, FORCE_CONSTANTS_2ND/FORCE_CONSTANTS, FORCE_CONSTANTS_3RD
-        - shengbte-qe: CONTROL, POSCAR, espresso.ifc2, FORCE_CONSTANTS_3RD
-        - hiphive: atom_prim.xyz, replicated_atoms.xyz, model2.fcs, model3.fcs
-
-        Parameters
-        ----------
-        folder : str
-            Chosen folder to load in system information.
-        supercell : (int, int, int), optional
-            Number of unit cells in each cartesian direction replicated to form the input structure.
-            Default is (1, 1, 1)
-        format : 'numpy', 'eskm', 'lammps', 'shengbte', 'shengbte-qe', 'hiphive'
-            Format of force constant information being loaded into ForceConstants object.
-            Default is ``'numpy'``
-        third_energy_threshold : float, optional
-            When importing sparse third order force constant matrices, energies below
-            the threshold value in magnitude are ignored. Units: eV/Angstrom^3
-            Default is None
-        distance_threshold : float, optional
-            When calculating force constants, contributions from atoms further than the
-            distance threshold will be ignored.
-        third_supercell : (int, int, int), optional
-            Takes in the unit cell for the third order force constant matrix.
-            Default is self.supercell
-        is_acoustic_sum : Bool, optional
-            If true, the acoustic sum rule is applied to the dynamical matrix.
-            Default is False
-        chunk_size : int, optional
-            Number of entries to process per chunk when reading sparse third order files.
-            Larger values use more memory but may be faster for very large files.
-            Default: 100000
-
-        Returns
-        -------
-        forceconstants: ForceConstants object
-            A new instance of the ForceConstants class
-        """
-        supercell = _normalize_supercell(supercell)
+                    chunk_size: int = 100000,
+                    *,
+                    second_format: str | None = None,
```

Comment:
This is good. I wonder if we should default it to any format.
Does it break previous examples?